### ISSUE TITLE
Channel Updating

### ISSUE THREAD

**Issue by vgztv** [justintv/Twitch-API#8]
Reposting from the Twitch support forums as I was post this here instead.

Attempting to update a channel's "status" and "game" as outlined here does not behave as expected.  While the returned JSON suggests both have been changed, only the current game actually changes.  Additionally, while further requests to the API reflect this change, the page itself, http://twitch.tv/:channel, does not.  It only actually updates in the dashboard, where you can force it to update, defeating the purpose of having API functionality for it altogether.

tl;dr, your https://api.twitch.tv/kraken/channels/:channel PUT functionality does not work.
